spark-3.1-spanner-lib/src/main/java/com/google/cloud/spark/spanner/SpannerDataReader_java

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.cloud.spark.spanner;
+
+import org.apache.spark.sql.sources.v2.reader.DataReader;
+
+public class SpannerDataReader<T> implements DataReader<T> {
```

Comment:
This is not even a java file. Is this added by mistake ?

Reply:
It was a java file but I removed the extension, I did intend to remove it.